rust/src/data/ffi.rs

```diff
@@ -290,11 +325,72 @@ pub extern "C" fn opendp_data__object_as_slice(obj: *const AnyObject) -> FfiResu
         util::into_raw(map);
         Ok(map_slice)
     }
+
+    fn lazyframe_to_raw(obj: &AnyObject) -> Fallible<FfiSlice> {
+        // Rechunk aggregates all chunks to a contiguous array of memory.
```

Comment:
In `concrete_series_to_raw` I see a call to `rechunk`, but I don't see it here. Is there an indirect call? Why mention here?

Reply:
No need to mention it, dropped the comment.